cmd/ebrelayer/main.go

```diff
@@ -370,6 +381,26 @@ func AddRelayerFlagsToCmd(cmd *cobra.Command) {
 		int32(oracleTypes.NetworkDescriptor_NETWORK_DESCRIPTOR_ETHEREUM),
 		"The network descriptor for the chain",
 	)
+	cmd.Flags().String(
```

Comment:
Are these flags being marked required or does the fetch and check error handle that automatically?

Reply:
I didn't find any parameter to set the flag as mandatory, and cobra will check the validatity.